helm/prometheus-rules/templates/alerting-rules/capi.rules.yml

```diff
@@ -61,4 +61,4 @@ spec:
           annotations:
             description: |-
               {{`Cluster {{$labels.exported_namespace}}/{{$labels.cluster_name}} is in a non healthy phase.`}}
-{{- end }}
+{{- end }}
```

Comment:
Why removing this EOF?

Reply:
happens due rebasing -- reverted the removal